src/pages/AdminDashboard/AdminInvestment.tsx

```
@@ -2,12 +2,19 @@
 import { Table } from "../../components/Table";
-import { FaRegArrowAltCircleRight } from "react-icons/fa";
-// import { FaRegArrowAltCircleRight } from "react-icons/fa";
+import { FaRegArrowAltCircleRight, FaTimesCircle } from "react-icons/fa";
+import Modal from "../../components/Modal";
+import PromptsCard from "../../components/PromptsCard";
+import { useFetchAllInvestmentsQuery } from "../../slices/investmentSlice";
 
 const AdminInvestment = () => {
    const [activeTab, setActiveTab] = useState("investment1");
-   const [selectedItem,setSelectedItem] = useState({})
+   const {data} = useFetchAllInvestmentsQuery({}) as any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)